python/llm/src/ipex_llm/transformers/npu_model.py

```diff
@@ -45,6 +45,9 @@ def ignore_argument(kwargs: dict, key: "str"):
 
 
 def save_low_bit(self, model_dir: str, *args, **kwargs):
+    if hasattr(self, "save_directory"):
+        logger.info(f"Model has already saved to {self.save_directory}.")
+        return 1
```

Comment:
What if "save_directory" is different from "model_dir"?

Reply:
> What if "save_directory" is different from "model_dir"?

Have updated the logic of save.

For all cases with `optimize_model=True`, specify `save_directory` is required during the first time to load model, ckpt is saved to `save_directory` after converting.